superchain/configs/mainnet/superchain.toml

```diff
@@ -8,7 +8,8 @@ delta_time =   1708560000 # Thu 22 Feb 2024 00:00:00 UTC
 ecotone_time = 1710374401 # Thu 14 Mar 2024 00:00:01 UTC
 fjord_time =   1720627201 # Wed 10 Jul 2024 16:00:01 UTC
 granite_time = 1726070401 # Wed 11 Sep 2024 16:00:01 UTC
-holocene_time = 1736445601 # Thu 09 Jan 2025 18:00:01 UTC
+holocene_time = 1736445601 # Thu 9 Jan 2025 18:00:01 UTC
```

Comment:
I just removed the `0` in the comment here to align it with the other comments.

Reply:
You may need to run some codegen again, looks like a test is failing.

Reply:
I think that's just the test that fails to indicate that chain configs changed, which is intended

Reply:
Or maybe not. It seems the output of the failed test is just truncated. This check isn't supposed to fail, in the sepolia Isthmus PR (https://github.com/ethereum-optimism/superchain-registry/pull/956) it didn't fail.

Reply:
It seems https://github.com/ethereum-optimism/superchain-registry/pull/951/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47 introduced a bug in the CCI check, it fails silently at the point where the first loop hits the `superchain.toml` file. Trying to fix it in https://github.com/ethereum-optimism/superchain-registry/pull/972/commits/446c3141f68038c7ce5824cc3aa0e753565eb5fe

Reply:
Nice, seems to work 🎉